module/zfs/dsl_scan.c

```diff
@@ -2184,52 +2184,75 @@ dsl_scan_ds_clone_swapped(dsl_dataset_t *ds1, dsl_dataset_t *ds2, dmu_tx_t *tx)
 {
```

Comment:
while we're here can you change the comment above the function to say `origin` instead of `parent`?

Reply:
I think that a simple s/parent/origin/ would not produce a good result.
Also, it seems that there is such a term as "clone parent-child dependency relationship", so "parent" is not completely out of place. And an unqualified "origin" seems to mean an origin snapshot.
Anyway, "parent" is more ambiguous than "origin" in this context, so I'll try to improve that comment.